api/alert_rules.go

```diff
@@ -0,0 +1,201 @@
+//
+// Author:: Darren Murray (<darren.murray@lacework.net>)
+// Copyright:: Copyright 2021, Lacework Inc.
+// License:: Apache License, Version 2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package api
+
+import (
+	"fmt"
+
+	"github.com/pkg/errors"
+)
+
+// AlertRulesService is the service that interacts with
+// the AlertRules schema from the Lacework APIv2 Server
+type AlertRulesService struct {
+	client *Client
+}
+
+type alertRuleType string
+
+type alertRuleSeverity int
+
+type AlertRuleSeverities []alertRuleSeverity
+
+// Alert Rule type is an enum in api v2, with only 1 supported value "Event"
```

Comment:
why? are we going to have more? 🤔 This seems like a lot of overhead for a hardcoded type.